_includes/configuration/advtable_value_series.md

```diff
@@ -0,0 +1,89 @@
+## `advtable_value_series`
+{{ site.requires_5_9v }}
+
+{% if pluginname != "Advanced Tables" %}
+> **Note**: The `advtable_value_series` option requires the Advanced Tables plugin.
+{% endif %}
+
+The `advtable_value_series` option configures value series for populating cells in a table. This option can be used to create row identifiers.
+
+**Type:** `Object`
+
+**Default Value:**
+
+```js
+{
+  // Natural number series
+  numeric: { 
+    title: 'Numeric',
+    update: true,
+    resizable: false,
+    generator: `GeneratorFunction` // For details, see: 'Creating a value series generator'
+  },
+  // English alphabetic series
+  alpha: {
+    title: 'Alpha',
+    update: true,
+    resizable: false,
+    generator: `GeneratorFunction` // For details, see: 'Creating a value series generator'
+  },
+}
+```
+
+Both default series are configured to update on table changes and not resize when using the resize bars.
+
+### Series configuration
```

Comment:
Should we explain that the "name" of the series column is the key for this object?

Reply:
Good point. I'll try and draft something up.

Reply:
Does something like this seem suitable?
`Each key in the 'advtable_value_series' option object indicates the name of the value series and its configuration.`

Reply:
Have added something very similar above the 'Series configuration' heading. Hopefully, it is clearer now.